include/fastad_bits/mat.hpp

```diff
@@ -1,16 +1,18 @@
 #pragma once
 #include <iostream>
+#include <iomanip>
 #include <vector>
 #include <algorithm>
 
+static inline constexpr unsigned int PRINT_WIDTH = 13;
```

Comment:
Put these inside `Mat` under `private`. I think this should be exclusive information for `Mat` and hence in private section. It should still be `static constexpr` because this value must be same across all `Mat` objects (`static`) and this value should not be modified/are known during compile-time (`constexpr`).